### ISSUE TITLE
not sort based on scores?

### ISSUE THREAD

**Issue by applepieiris** [jhj0517/sam2-playground#35]
hi I see in the latest code using sam2.1 model, the prediction results are not sorted based on scores just like this :
`predicted_masks, scores, logits = self.predict_image(
                image=image,
                model_type=model_type,
                box=box,
                point_coords=point_coords,
                point_labels=point_labels,
                multimask_output=hparams["multimask_output"],
                invert_mask=invert_mask
            )
            generated_masks = self.format_to_auto_result(predicted_masks)`

in function `format_to_auto_result` , there is no sorting as well. I think we should sort the mask based on the score or display all the masking results in frontend UI
